app/server.go

```diff
@@ -365,7 +365,7 @@ func NewServer(options ...Option) (*Server, error) {
 		}
 	}
 
-	if htmlTemplateWatcher, err2 := utils.NewHTMLTemplateWatcher("templates"); err2 != nil {
+	if htmlTemplateWatcher, err2 := utils.NewHTMLTemplateWatcher("templates", "templates/compiled"); err2 != nil {
```

Comment:
some people relay on modified version of the template files in their own systems, so this this changes of routes can change unexpectedly the behavior for them. Can we maintain the same paths for the generated templates?

Reply:
Yes done. But now this requires us to mention each generated html file in `.gitignore`. In future when all email templates have migrated to mjml then we can simplify this with `templates/*.html`

Reply:
Yes, that is ok for now.